devel/universal-ctags/Portfile

```diff
@@ -3,10 +3,10 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        universal-ctags ctags feffe43a4a9d339cd6701744589746e2297e4567
+github.setup        universal-ctags ctags 5.9.20201227.0 p
 name                universal-ctags
-version             20200111
-revision            1
+epoch               2
```

Comment:
Why is it necessary to set epoch to 2? According to docs it's 0 by default and only needs to be increased by 1.

Reply:
Sorry, the wrong number entered.